src/index.ts

```diff
@@ -296,6 +304,24 @@ export const createInstance = (config: Config): Instance => {
         const tx = await zAuctionInstance.buyNow(params, signer);
         return tx;
       },
+      getBuyNowPrice: async (
+        tokenId: string,
+        signer: ethers.Signer
+      ): Promise<number> => {
+        const zAuctionInstance = await getZAuctionInstanceForDomain(
+          tokenId,
+          config.zAuctionRoutes,
+          zAuctionRouteUriToInstance,
+          domainIdToDomainName
+        );
+        const listing = await zAuctionInstance.getBuyNowPrice(tokenId, signer);
```

Comment:
Check if `listing.holder === owner of the domain` if it is, return listing.price, if not return 0